### DISCUSSION TITLE
Redelivery

### DISCUSSION THREAD

**Discussion by xoxoedg** [arkmq-org/arkmq-org-broker-operator#629] [Q&A]
1. Is it possible to configure delayed redelivery as shown in the artemis? 
    docs(https://activemq.apache.org/components/artemis/documentation/1.0.0/undelivered-messages.html)

2. Is there a full view on configurations for the ActiveMQArtemisAddress kind?

**Comment by gaohoward** [comment]
The best approach is to use brokerProperties in broker CR to configure it.
Not sure I understand the second question, but the ActiveMQArtemisAddress doesn't support the configuration.

**Reply by xoxoedg** [reply]
Hi sorry, what do you mean with CR and thanks for the reply

**Comment by gaohoward** [comment]
I mean the ActiveMQArtemis CR (check out its crd [here](https://github.com/artemiscloud/activemq-artemis-operator/blob/main/deploy/crds/broker_activemqartemis_crd.yaml#L354). And you can get more details in this doc:
https://github.com/artemiscloud/activemq-artemis-operator/blob/main/docs/help/operator.md#configuring-brokerproperties
